### DISCUSSION TITLE
Connecting an external audio track

### DISCUSSION THREAD

**Discussion by Qvatom** [gerbera/gerbera#3112] [Q&A]
I have a file Name.mkv, next to it is a file Name.mka with audio tracks.
The fact that Gerbera cannot send them to the device at the same time has already been understood.  But can Gerbera see a file with external audio tracks and include them in the stream if I want to ttranscode the file.  Is there any way at all to connect audio tracks from an external file?
Thank you advance

**Comment by KarlStraussberger** [comment]
This could be easily done by creating a small shell script that is looking for the files and relaying all its work to vlc or ffmpeg in the end.
